core/src/test/resources/sql/some.iq

```diff
@@ -847,5 +847,92 @@ where sal > all (select comm from "scott".emp where comm is not null);
 
 !ok
 
+# [CALCITE-4758] When SOME sub-query is SqlNodeList, Calcite returns wrong result
+
+# make sure the SqlNodeList converted to VALUES not OR condition
+
```

Comment:
this empty line makes it less clear that the comment applies to the threshold setting